frontend/src/metabase/visualizer/hooks/use-can-handle-active-item.unit.spec.tsx

```diff
@@ -0,0 +1,287 @@
+import type { Active } from "@dnd-kit/core";
+import { renderHook } from "@testing-library/react";
+import type { PropsWithChildren } from "react";
+
+import { MetabaseReduxProvider } from "metabase/lib/redux";
+import { mainReducers } from "metabase/reducers-main";
+import { getStore } from "metabase/store";
+import {
+  getHoveredItems,
+  getReferencedColumns,
+} from "metabase/visualizer/selectors";
+import { isDraggedColumnItem } from "metabase/visualizer/utils";
+import type { DatasetColumn, VisualizerDataSource } from "metabase-types/api";
+import { createMockColumn } from "metabase-types/api/mocks";
+import { createMockState } from "metabase-types/store/mocks";
+
+import { useCanHandleActiveItem } from "./use-can-handle-active-item";
+
+// Mock the selectors
+jest.mock("metabase/visualizer/selectors", () => ({
```

Comment:
We don't usually mock selectors in tests, do you think you could add mock data to the store instead?